### ISSUE TITLE
Using OpenALPR for Sudan plates

### ISSUE THREAD

**Issue by mohamed-moaz** [openalpr/openalpr#410]
I downloaded the windows release (openalpr-2.3.0-win-64bit.zip) and i am wondering if OpenALPR can be used to detect Sudan plates which have arabic characters, i have the tesseract arabic trained data (ara.traineddata) in runtime_data\ocr\tessdata directory.
How can i modify the config file(s) to detect and recognize the plate (since i can't train the detector, i think the 'us' plates are much similar to the Sudanese plates) ?

![sudan](https://cloud.githubusercontent.com/assets/22828889/19398346/e497c58a-9254-11e6-8232-c54e4bf736ae.jpg)

**Comment by matthill** [comment]
Hi Mohamed,

OpenALPR doesn't have any training data for Sudan plates.  Even when properly trained, it will also need code changes to work well.  These plates look tricky, since they're both multi-row and multi-column.  

In the example above, which sections of the plate are most important?

**Comment by mohamed-moaz** [comment]
Hi matthill, thanks for replying,

The lower right section is the most important (the one with the numbers 5490), the lower left section is for the state number, and the upper section is fixed for all Sudanese plates.

![14658212_10207576968925039_1778226536_n](https://cloud.githubusercontent.com/assets/22828889/19415665/ca2f2432-937f-11e6-88b2-5d096013f2fc.jpg)
![14658154_10207576967685008_1050791673_n](https://cloud.githubusercontent.com/assets/22828889/19415667/d72dbc20-937f-11e6-9211-9fc1d9f1c4ef.jpg)

**Comment by matthill** [comment]
Yes, it may be possible to read that then... The trick will be removing the left part of the plate.  I believe code modifications are still necessary.

**Comment by mohamed-moaz** [comment]
What are the necessarily modifications in the code or in the configuration file to remove the left part of the plate ?

**Comment by makkimadani** [comment]
> What are the necessarily modifications in the code or in the configuration file to remove the left part of the plate ?

المشكلة اتحلت عندك ؟ بتواجهني نفس المشكلة
